cmd/podman/create.go

```diff
@@ -388,12 +396,21 @@ func parseCreateOpts(ctx context.Context, c *cli.Context, runtime *libpod.Runtim
 		return nil, errors.Errorf("--cpu-quota and --cpus cannot be set together")
 	}
 
-	utsMode := container.UTSMode(c.String("uts"))
-	if !cc.IsNS(string(utsMode)) && !utsMode.Valid() {
+	utsModeStr := c.String("uts")
+	if !c.IsSet("uts") && c.IsSet("pod") {
```

Comment:
For these - we might want to do a query against the pod to see if it's sharing the namespace, and only set pod mode if it is.